components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/dao/OrganizationManagementDAO.java

```diff
@@ -446,4 +448,17 @@ default List<String> getOrganizationsMetaAttributes(boolean recursive, Integer l
         throw new OrganizationManagementServerException("getOrganizationsMetaAttributes is not implemented in "
                 + this.getClass().getName());
     }
+
+    /**
+     * Retrieve a map of organization IDs to their corresponding organization names.
+     *
+     * @param orgIds The list of organization IDs to fetch names for.
+     * @return A map where the key is the organization ID and the value is the organization name.
+     * @throws OrganizationManagementException If an error occurs while retrieving organization names.
+     */
+    default Map<String, String> getOrganizationNamesByIds(List<String> orgIds) throws OrganizationManagementException {
```

Comment:
Since this is an internal interface and already implemented cachebackedDAO and DAOImpl, do we need to have the default implementation in here ?